src/resources/common/Platform.ts

```diff
@@ -12,7 +12,20 @@ type CheckoutConfiguration = {
     ecProducts: { enabled: boolean };
 };
 
+export enum TransferMatchAmount {
+    MAXIMUM = "maximum",
+    MINIMUM = "minimum",
+    EXACT = "exact",
+}
+
+export type BankTransferConfiguration = {
+    enabled: boolean;
+    matchAmount: TransferMatchAmount;
+    expiration: string;
```

Comment:
hmm... i'd call this `expireDate` to better describe the `string`, but not a big deal 😄

Reply:
Let me double check the type here 🙂

Reply:
Got the fixture wrong. It seems to be the expiration period. Somehow...

Reply:
oh nice! thanks for the indirect fix XD